test/metabase/api/user_test.clj

```diff
@@ -147,11 +175,59 @@
                   (map (partial merge user-defaults))
                   (map #(dissoc % :is_qbnewb :last_login)))
              (->> (mt/user-http-request :crowberto :get 200 "user", :include_deactivated true)
+                  (filter mt/test-user?)
+                  group-ids->sets
+                  mt/boolean-ids-and-timestamps
+                  (map #(dissoc % :is_qbnewb :last_login)))))
+      (is (= (->> [{:email                  "trashbird@metabase.com"
```

Comment:
nit: some of these maps could probably be merged in from `metabase.test.data.users`, but some of the keys would need renamed, so not sure if worth the trouble.

Reply:
I dislike test data with lots of indirection because tests are the place to go for concrete instantiations of stuff and concrete calls and stuff like that, imo

Reply:
Sure, makes sense.